src/main/java/duke/command/CommandFind.java

```diff
@@ -0,0 +1,33 @@
+package duke.command;
+
+import duke.util.Storage;
+import duke.util.TaskList;
+import duke.util.Ui;
+
+/**
+ * The class that models a find command.
+ */
+public class CommandFind extends Command {
+    private String keyword;
+
+    public CommandFind(String keyword) {
+        this.keyword = keyword;
+    }
+
+    /**
+     * Execute find action and then print result message.
+     * @param tasks The taskList to search keywork in.
+     * @param ui The Ui object to print messages.
+     * @param storage The Storage object that auto-saves after modification.
+     */
+    @Override
+    public void execute(TaskList tasks, Ui ui, Storage storage) {
+        TaskList matched = tasks.find(keyword);
```

Comment:
Should this variable be renamed to `matchedTasks` for better identification?

Reply:
Thanks for the suggestion!